src/ImageSharp.Web/Middleware/ImageSharpMiddleware.cs

```diff
@@ -32,13 +31,13 @@ public class ImageSharpMiddleware
         /// Used to temporarily store source metadata reads to reduce the overhead of cache lookups.
         /// </summary>
         private static readonly ConcurrentTLruCache<string, ImageMetadata> SourceMetadataLru
-            = new(1024, TimeSpan.FromMinutes(5));
+            = new(1024, TimeSpan.FromSeconds(30));
 
         /// <summary>
         /// Used to temporarily store cache resolver reads to reduce the overhead of cache lookups.
         /// </summary>
         private static readonly ConcurrentTLruCache<string, (IImageCacheResolver, ImageCacheMetadata)> CacheResolverLru
-            = new(1024, TimeSpan.FromMinutes(5));
+            = new(1024, TimeSpan.FromSeconds(30));
```

Comment:
are you going to pop these back up? or make them configurable?

Reply:
I guess we could make them configurable but I've no idea what to name the properties (_InMemoryCacheDuration_?)

I found that if I deleted the cache folder when something was stored in there it prompted a null reference exception in the cache resolver. I dropped the times to make that less likely.

Reply:
Ah... They're private static. We can't pass options.

Reply:
> Ah... They're private static. We can't pass options.

True, there might be an argument for making them non static, for performance.
In our scenario, we have multiple instances of the middleware (per tenant), so multiple instances of the cache per tenant, is perhaps desirable, as the items count will be less, so the lookup a little faster.
(then becomes configurable)

Something we can think about later though :)

Reply:
Yeah that might well be an idea then.